OMGVA PoS/Data layer/Repositories/Tax/TaxRepository.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.EntityFrameworkCore;
+using OMGVA_PoS.Data_layer.Context;
+
+namespace OMGVA_PoS.Data_layer.Repositories.Tax;
+
+public class TaxRepository(OMGVADbContext context) : ITaxRepository
```

Comment:
I would suggest assigning context to a private readonly variable and then using it. This would be a better practice because repository classes repeatedly use the injected dependencies. It would also help avoid accidental reassignments.

Reply:
Thanks will do